hs/Database/Tables.hs

```diff
@@ -70,44 +66,39 @@ Distribution
     description String
     deriving Show
 
-Graphs
+Graph
     gId Int
     title String
     deriving Show
 
-Rects
-    gId Int
+Text
+--    gId Int
     rId String
-    width Rational
-    height Rational
-    xPos Rational
-    yPos Rational
+    coord Point
+    text String
+    deriving Show
+
+Shape
+    id_ String
+    coord Point
+    width Double
+    height Double
     fill String
     stroke String
+    text [Text]
     isHybrid Bool
-    deriving Show
+    tolerance Double
+    isBool Bool
```

Comment:
Why don't we use a `type` string instead of separate attributes for hybrids and bools?